WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -1161,6 +1177,62 @@
 			);
 			target = 3F7AE0B42D9B30A100AB4892 /* WordPressData */;
 		};
+		4ABC59762DE52AA2005A6B84 /* Exceptions for "Classes" folder in "WordPressDraftActionExtension" target */ = {
+			isa = PBXFileSystemSynchronizedBuildFileExceptionSet;
+			membershipExceptions = (
+				"System/ApiCredentials+BuildSecrets.swift",
```

Comment:
As a future improvement, we could consider moving the `ApiCredentials` `struct` to BuildSettingsKit, so it's available via `import`.